CPT331.WebAPI/App_Start/SwaggerConfig.cs

```diff
@@ -0,0 +1,246 @@
+using System.Web.Http;
+using WebActivatorEx;
+using CPT331.WebAPI;
+using Swashbuckle.Application;
+
+[assembly: PreApplicationStartMethod(typeof(SwaggerConfig), "Register")]
+
```

Comment:
What are the advantages of using this over use Postman? Testing/tooling related code would be better off in a different project.